test/utils/constants.ts

```diff
@@ -1,2 +1,4 @@
 export const PASSWORD = "e45432jklfdsab"
 export const workspaceDir = "workspaces"
+export const REVERSE_PROXY_BASE_PATH = "ide"
+export const REVERSE_PROXY_PORT = "8000"
```

Comment:
Should we go ahead and make them configurable with environment variables?  Might be nice so you can run the proxy locally on whatever port/path you want.  I am not sure what we would want to call them but maybe something like this:

```suggestion
export const REVERSE_PROXY_BASE_PATH = process.env.CS_TEST_REVERSE_PROXY_BASE_PATH || "ide"
export const REVERSE_PROXY_PORT = process.env.CS_TEST_REVERSE_PROXY_PORT || "8000"
```

Reply:
Easy enough!